tests/console_test.go

```diff
@@ -133,7 +133,7 @@ var _ = Describe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@redha
 				},
 					table.Entry("with ContainerDisk", newVirtualMachineInstanceWithAlpineContainerDisk),
 					table.Entry("with OCS Filesystem Disk", newVirtualMachineInstanceWithAlpineOCSFileDisk),
-					table.Entry("with OCS Block Disk", newVirtualMachineInstanceWithAlpineOCSBlockDisk),
+					table.FEntry("with OCS Block Disk", newVirtualMachineInstanceWithAlpineOCSBlockDisk),
```

Comment:
`FEntry` -> `Entry`
the same relevant for all focused tests :)

Reply:
Done